### PR TITLE
Minor 1.7

### PR DESCRIPTION
**Pull request by XFox111**
Implements following issues: #29, #27

## Changelog
- Complete UI/UX overhaul
- Updated default hotkeys (#27)
- Added option to disable deletion confirmation (#29)
- Added Chinese translation
- Fixed font artifacts on Windows

## PR Checklist
- [x] Change extension version in the manifest

### CODE REVIEW

File: _locales/zh-CN/messages.json [hunk 1]
```diff
@@ -19,6 +19,11 @@
 		"message": "设置",
 		"description": "Alternative text for options button in the pane"
 	},
+	"closePanel":
+	{
+		"message": "关",
+		"description": "Alternative text for close panel button"
+	},
```
Comment (XFox111) [inline _locales/zh-CN/messages.json:26]: @Dustin-Jiang Hi. Could you please check if this translation is correct? I've also changed a string on [line 124](https://github.com/XFox111/ChromiumTabsAside/pull/31/commits/700d3e775ed9af3e614e366537106a22c64d2594#diff-7ee29da7f716126a2b51a26b56500809R124)

**Comment by XFox111** [comment]
@Dustin-Jiang I still need your response

File: _locales/zh-CN/messages.json (hunk 1 above)
Comment (Dustin-Jiang) [reply _locales/zh-CN/messages.json:26]: OK. I think it should be translated as '关闭‘ or '关闭侧栏', which is better and closer to native speaking.

Comment (Dustin-Jiang) [reply _locales/zh-CN/messages.json:26]: The string on line 124 is correct.

**Pull request merged**